pdr_backend/trueval/trueval.py

```diff
@@ -18,11 +18,14 @@
   You need to change the code to support more complex flows. Now, it's based on ccxt
```

Comment:
You can move trueval.py's functionality into trueval/main.py, then delete trueval.py